environment-setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh -ex
+
+# Prepare a development environment for running telepresence and its test
+# suite.  These steps should typically only be required once to prepare the
+# environment.
+
+PROJECT_NAME=$1
+CLUSTER_NAME=$2
+CLOUDSDK_COMPUTE_ZONE=$3
+
+case "$(uname -s)" in
+     Darwin)
+	 brew update > /dev/null
+	 brew cask install osxfuse
+	 brew install python3 sshfs
+	 ;;
+
+     Linux)
+	 sudo apt install sshfs conntrack
+	 ;;
+     *)
+	 echo "Unknown platform."
+	 exit 1
+esac
+
+# Record some debugging info:
+python --version
+python2 --version
+python3 --version
+ruby --version
+
+# Make sure gcloud is installed.  This includes kubectl.
+./ci/setup-gcloud.sh "${PROJECT_NAME}" "${CLUSTER_NAME}" "${CLOUDSDK_COMPUTE_ZONE}"
+
+# Make sure torsocks is installed:
+./ci/build-torsocks.sh
```

Comment:
It's not trivial to get away from this step (which installs torsocks from source on Linux).  When I replace this with `apt install torsocks` this failure results:

```
 17.5 28 | 15:06:14 libtorsocks(11202): Malformed configuration pair on line 3 in configuration file, "AllowInbound 1"
  17.5 28 | 15:06:14 libtorsocks(11202): Malformed configuration pair on line 5 in configuration file, "AllowOutboundLocalhost 1"
  17.5 28 | 15:06:14 libtorsocks(11202): Malformed configuration pair on line 7 in configuration file, "TorPort 46193"
  17.5 28 | 15:06:14 libtorsocks(11202): WARNING: The symbol res_query() was not found in any shared library with the reported error: Not Found!
  17.5 28 |   Also, we failed to find the symbol __res_query() with the reported error: Not Found
  17.5 28 | 15:06:14 libtorsocks(11202): WARNING: The symbol res_search() was not found in any shared library with the reported error: Not Found!
  17.5 28 |   Also, we failed to find the symbol __res_search() with the reported error: Not Found
  17.5 28 | 15:06:14 libtorsocks(11202): WARNING: The symbol res_send() was not found in any shared library with the reported error: Not Found!
  17.5 28 |   Also, we failed to find the symbol __res_send() with the reported error: Not Found
  17.5 28 | 15:06:14 libtorsocks(11202): WARNING: The symbol res_querydomain() was not found in any shared library with the reported error: Not Found!
  17.5 28 |   Also, we failed to find the symbol __res_querydomain() with the reported error: Not Found
```

Apart from the superficial interpretation, I'm not sure what this is indicating yet.